node/impl/full/sync.go

```diff
@@ -105,6 +106,26 @@ func (a *SyncAPI) SyncIncomingBlocks(ctx context.Context) (<-chan *types.BlockHe
 	return a.Syncer.IncomingBlocks(ctx)
 }
 
+func (a *SyncAPI) SlashFilterMinedBlock(ctx context.Context, bh *types.BlockHeader) (*types.BlockHeader, error) {
+	parent, err := a.Syncer.ChainStore().GetBlock(ctx, bh.Parents[0])
+	if err != nil {
+		return nil, nil
+	}
+	err = a.SlashFilter.MinedBlock(ctx, bh, parent.Height)
```

Comment:
Is the SlashFilter always initiated, or could it be `nil`?

Reply:
It shouldn't be possible for nil